### PR TITLE
feat(demo): add sticky bottom pagination to the card demo

### LINKED ISSUE #4884: Add pagination footer to React Card view demo
Update the demo here to include a pagination control in the sticky page footer: https://www.patternfly.org/v4/components/card/react-demos. Core issue here: https://github.com/patternfly/patternfly/issues/3225 The pagination should be functional and work in sync with the toolbar pagination above the card grid.

### PR DESCRIPTION
**Pull request by boaz0**
**What**: Closes #4884 

//cc @mcarrano to see this go to "Demo" and choose the primary detail card view demo.

### CODE REVIEW

**Comment by tlabaj** [comment]
@Katie.  It looks like this partial resolves #4962

**Review by mcarrano — CHANGES_REQUESTED** [review]
@boaz0 The pagination footer should span the entire width of the content area as shown in the core demo here: https://www.patternfly.org/v4/components/card/html-demos/card-view/ Can we also add this to the React card view demo so we have parity with core? Thanks.

File: packages/react-core/src/components/Pagination/Pagination.tsx
Comment (mcoker) [inline packages/react-core/src/components/Pagination/Pagination.tsx:83]: It can also be used with top pagination if the pagination is on its own and not in a toolbar.

**Comment by mcoker** [comment]
The demo for card view bottom pagination should be implemented either as part of https://github.com/patternfly/patternfly-react/issues/4962, or after that PR is merged. The demo uses a sticky page section with pagination in the section, which gives the full width layout in the page main section.

Screen Shot 2020-10-12 at 10 04 08 AM

**Review by mcoker — COMMENTED** [review]
Code LGTM! In core we have sticky examples with content to show the sticky behavior. Should we add those here, too? https://www.patternfly.org/v4/components/pagination/html

Also do we want to see if @kmcfaul can update the primary-detail and card component card view demos as part of https://github.com/patternfly/patternfly-react/issues/4962 or should we create a separate issue to update those demos after https://github.com/patternfly/patternfly-react/issues/4962 is merged?

**Review by redallen — APPROVED** [review]
There's a race between this and #4977 for `Pagination.tsx`. If you could add a `<div style={{ height: '100vh' }}` somewhere that'd go a long way to demoing that it's sticky.

**Review by redallen — CHANGES_REQUESTED** [review]
Meant to request changes, not approve, sorry!

**Comment by boaz0** [comment]
@redallen sure. I will do that soon. Thanks.

**Comment by boaz0** [comment]
@mcarrano @mcoker & @redallen thanks for your feedback and review.
I figured out what you meant and updated the example.
Can you please review this again? Thanks again for the help and support.

**Review by redallen — APPROVED** [review]
Thanks for your support @boaz0 !

**Comment by mcarrano** [comment]
Primary-detail demo looks great @boaz0 . Can we also add this footer to the Card view demo here? https://patternfly-react-pr-5000.surge.sh/components/card/react-demos

File: packages/react-core/src/components/Page/PageSection.tsx
Comment (mcoker) [inline packages/react-core/src/components/Page/PageSection.tsx:38]: I think this is the same as @kmcfaul's PR? Or is this something different? https://github.com/patternfly/patternfly-react/pull/5025/files

Comment (boaz0) [reply packages/react-core/src/components/Page/PageSection.tsx:38]: Oh yep, I can copy the relevant code or we can wait on @kmcfaul changes to be merged and I will do rebase.
Whatever suites you.

Comment (mcoker) [reply packages/react-core/src/components/Page/PageSection.tsx:38]: Looks like #5025 was just merged, so you could use those props to fix these demos now! Thanks @boaz0!!

Comment (boaz0) [reply packages/react-core/src/components/Page/PageSection.tsx:38]: Sweet thanks!!!

**Review by mcoker — CHANGES_REQUESTED** [review]
Primary detail looks great, but the card component card view demo isn't rendering. @redallen mentioned it's because of this

>  defaultVendors~main.cf2d97b1.bundle.js:45 ReferenceError: page is not defined

**Comment by boaz0** [comment]
@mcoker DONE thanks to @redallen help as usual :bow:

**Review by redallen — DISMISSED** [review]

File: packages/react-core/src/demos/Card/Card.md
Comment (mcoker) [inline packages/react-core/src/demos/Card/Card.md:748]: Can you add `isFilled` to the section with the cards?

Comment (boaz0) [reply packages/react-core/src/demos/Card/Card.md:748]: good catch

**Review by mcoker — APPROVED** [review]
LGTM!!! 🥳

**Review by tlabaj — APPROVED** [review]
LGTM

**Review by mcarrano — APPROVED** [review]
Looks good now. Thanks @boaz0 !

**Review by redallen — APPROVED** [review]

**Comment by boaz0** [comment]
Thank you all

**Pull request merged**
